e2e/terraform/.terraform.lock.hcl

```diff
@@ -5,6 +5,7 @@ provider "registry.terraform.io/hashicorp/aws" {
   version = "3.55.0"
   hashes = [
     "h1:Ls8MD4Olzybw9n0mP5Lr1S2PnZzlSKrpxvYN9u2p/dM=",
+    "h1:pQzssRQ6BMdyO65Lq5PRx766l8swjcCNKrNyCkcJa6I=",
```

Comment:
Was this accidentally committed or are these real updates we need to the lockfile?

Reply:
Yikes I was sloppy with this commit. Sorry. :(